src/config/development.tsx

```diff
@@ -51,6 +51,7 @@ const CELO_NETWORK_NUMBER = 44787;
 const CLASSIC_NETWORK_NUMBER = 63;
 const SOLANA_NETWORK: NonEVMChain = {
 	id: 0,
+	networkId: 103,
```

Comment:
I am suspicious about using `networkId`
What if in the future an EVM chain comes with the same value?!

Reply:
What is your suggestion? we did this based on the naming of the backend.
I think if some day a chain has added and we decided to support it on Giveth, then we can change the name.
What do you think?

Reply:
I mean, you don't have to add anything here at this stage with your intended changes. So please don't modify it so far and let it for other PRs